src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -2204,6 +2270,28 @@ internal ScriptRequirements GetScriptRequirements()
             }
         }
 
+        private List<string> HandleRequiresOSVersionsArgument(Ast argumentAst, object arg, ref List<string> requiredOSVersions)
+        {
+            if (!(arg is string))
+            {
+                ReportError(argumentAst.Extent,
+                    nameof(ParserStrings.RequiresInvalidStringArgument),
+                    ParserStrings.RequiresInvalidStringArgument,
+                    osVersionsToken);
+            }
+            else
+            {
+                if (requiredOSVersions == null)
```

Comment:
Definitely use braces with the `if`. Some older code in the codebase doesn't, but ideally it should be changed as well.

Also, at this point, you know the list doesn't exist, so it can't contain the argument. So you can avoid redundant checks:

```csharp
private void HandleRequiresOSTypesArgument(string osName, ref List<string> requiredOSTypes)
{
    if (requiredOSTypes == null)
    {
        requiredOSTypes = new List<string>(){ osName };
        return;
    }

    // Need to check case-insensitively for OSName already being in the list here
    ...
}
```